synapse/lib/agenda.py

```diff
@@ -598,22 +598,33 @@ async def add(self, cdef):
         Persistently adds an appointment
 
         Args:
-            cdef (dict):  May contain the following keys:
-                creator (str):  iden of the creating user
-                iden (str): Iden of the appointment
-                storm (str): storm query to run
-                reqs (Union[None, Dict[TimeUnit, Union[int, Tuple[int]], List[...]):
+            cdef (dict):  Dictionary containing the Cron definition.
+
+        Notes:
+            The cron definition may contain the following keys:
+
+                creator (str)
+                    iden of the creating user.
```

Comment:
Should uppercase the first letter of 'Iden' to be consistent with the iden description